File: app/src/androidTest/java/org/secuso/privacyfriendlytodolist/ApplicationTest.java
package org.secuso.privacyfriendlytodolist;

import static junit.framework.TestCase.assertEquals;

import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
import androidx.test.platform.app.InstrumentationRegistry;

import org.junit.Test;
import org.junit.runner.RunWith;

/**
 * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
 */
@RunWith(AndroidJUnit4ClassRunner.class)
public class ApplicationTest {
    @Test
    public void testInstrumentationTest() throws Exception {
        assertEquals("org.secuso.privacyfriendlytodolist", InstrumentationRegistry.getInstrumentation().getTargetContext().getPackageName());
    }
}

File: app/src/androidTest/java/org/secuso/privacyfriendlytodolist/androidtest/ApplicationTest.kt
/*
Privacy Friendly To-Do List
Copyright (C) 2016-2024  Dominik Puellen

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <https://www.gnu.org/licenses/>.
*/
package org.secuso.privacyfriendlytodolist.androidtest

import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
import androidx.test.platform.app.InstrumentationRegistry
import junit.framework.TestCase
import org.junit.Test
import org.junit.runner.RunWith

/**
 * [Testing Fundamentals](http://d.android.com/tools/testing/testing_android.html)
 */
@RunWith(AndroidJUnit4ClassRunner::class)
class ApplicationTest {
    @Test
    @Throws(Exception::class)
    fun testInstrumentationTest() {
        TestCase.assertEquals(
            "org.secuso.privacyfriendlytodolist",
            InstrumentationRegistry.getInstrumentation().targetContext.packageName)
    }
}